include/eve/module/elliptic/regular/impl/ellint_1.hpp

```diff
@@ -86,17 +88,19 @@ ellint_1_(EVE_SUPPORTS(cpu_), T phi0, T x) noexcept requires has_native_abi_v<T>
 
 // -----------------------------------------------------------------------------------------------
 // Masked cases
-template<conditional_expr C, typename ... Ts>
+template<conditional_expr C, typename T0, typename ... Ts>
 EVE_FORCEINLINE auto
-ellint_1_(EVE_SUPPORTS(cpu_), C const& cond, Ts ... ts) noexcept
+ellint_1_(EVE_SUPPORTS(cpu_), C const& cond, T0 t0, Ts ... ts) noexcept
+-> decltype( if_else(cond, ellint_1(t0, ts...), t0) )
```

Comment:
are we confortable letting it this way or do you wish to have a traits for ?

Reply:
yes for me we are confortable